misc/update/nix/tmux/monitor.php

```diff
@@ -217,7 +217,7 @@
 						$stamp = 'UNIX_TIMESTAMP(MIN(dateadded))';
 
 						switch (true) {
-							case strpos($tbl, 'collections_') !== false:
+							case strpos($tbl, 'collections') !== false:
```

Comment:
Wouldn't it just be better to make the SQL query be something more descript than 'SHOW TABLE STATUS' and not worry about the specificity of the logic gates in that switch?  Something more like what we do with the TPG patch changes.

Honestly, the more I think about it, this whole procedure could likely be done with an SP and reduce the PHP code requirement altogether.